platform/android/export/export_plugin.cpp

```diff
@@ -2002,6 +2003,13 @@ Error EditorExportPlatformAndroid::run(const Ref<EditorExportPreset> &p_preset,
 		return ERR_UNCONFIGURED;
 	}
 
+	// Force device CPU architecture
+	String preset_arch = "architectures/" + devices[p_device].cpu_abi;
+	bool is_device_abi_enabled = p_preset->get(preset_arch);
+	if (!is_device_abi_enabled) {
+		p_preset->set(preset_arch, true);
+	}
```

Comment:
We should provide a warning to the user, maybe throw an error dialog with the option to continue regardless. I don't like the idea of forcing a behavior without user consent and / or knowledge.